### PR TITLE
Update start-translating-openedx.rst file

### PR DESCRIPTION
**Pull request by jesgreco**
Adding a section called "Useful Links"

### CODE REVIEW

**Comment by openedx-webhooks** [comment]
Thanks for the pull request, @jesgreco!  Please note that it may take us up to several weeks or months to complete a review and merge your PR.

Feel free to add as much of the following information to the ticket as you can:
- supporting documentation
- [Open edX discussion forum threads](https://discuss.openedx.org/)
- timeline information ("this must be merged by XX date", and why that is)
- partner information ("this is a course on edx.org")
- any other information that can help Product understand the context for the PR

All technical communication about the code itself will be done via the GitHub pull request interface. As a reminder, [our process documentation is here](http://edx-developer-guide.readthedocs.org/en/latest/process/overview.html).


Please let us know once your PR is ready for our review and all tests are green.


:warning: We can't start reviewing your pull request until you've submitted a     [signed contributor agreement](https://openedx.org/cla)     or indicated your institutional affiliation. Please see the [CONTRIBUTING](https://github.com/openedx/.github/blob/master/CONTRIBUTING.md)     file for more information.     If you've signed an agreement in the past, you may need to re-sign.  See     [The New Home of the Open edX Codebase](https://open.edx.org/blog/the-new-home-of-the-open-edx-codebase/)     for details.

    
Once you've signed the CLA, please allow 1 business day for it to be processed.  After this time, you can re-run the     CLA check by adding a comment here that you have signed it.  If the problem persists, you can tag the `@openedx/cla-problems` team in a     comment on your PR for further assistance.

File: source/translators/quickstarts/start-translating-openedx.rst [hunk 1]
```diff
@@ -101,3 +101,10 @@ Next Steps
   `docs.opened.org Github repository`_
 
 .. _docs.opened.org Github repository: https://github.com/openedx/edx-documentation
+
+Useful links
+**********
+
+	* Confluence site for Translation Working Group: https://openedx.atlassian.net/wiki/spaces/COMM/pages/3157524644/Translation+Working+Group
+	* Open edX WG website link: https://openedx.org/open-edx-community-working-groups/
```
Comment (feanil) [inline source/translators/quickstarts/start-translating-openedx.rst:109]:
```suggestion
* `Confluence site for Translation Working Group <https://openedx.atlassian.net/wiki/spaces/COMM/pages/3157524644/Translation+Working+Group>`_

* `Open edX WG website link <https://openedx.org/open-edx-community-working-groups/>`_
```

The bullet points shouldn't be tabbed in and we can make the links more standard RST links for cleanliness.

**Review by feanil — COMMENTED** [review]
Change generally looks good and we can merged once the suggestions are accepted and we've got a signed CLA.

**Review by feanil — APPROVED** [review]

**Comment by openedx-webhooks** [comment]
@jesgreco 🎉 Your pull request was merged!  Please take a moment  to [answer a two question survey](https://docs.google.com/forms/d/e/1FAIpQLSceJOyGJ6JOzfy6lyR3T7EW_71OWUnNQXp68Fymsk3MkNoSDg/viewform?usp=pp_url&entry.1671973413=openedx/docs.openedx.org&entry.867055334=https://github.com/openedx/docs.openedx.org/pull/415&entry.1484655318=https://github.com/jesgreco&entry.752974735=2023-10-26+13:27&entry.1917517419=2023-11-01+17:50&entry.2133058324=Yes) so we can improve your experience in the future.

**Pull request merged**